<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AC Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>277</width>
      <height>228</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Serial port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portName"/>
       </item>
       <item>
        <widget class="QPushButton" name="commButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="dumpAllPackets">
         <property name="text">
          <string>Dump raw communication</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="powerState">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Power</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current temperature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="currentTemp">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>23</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Set point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="setPoint">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>23</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modeSelector">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Cool</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Heat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dry</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fan</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Fan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fanSelector">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Eco</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Powerful</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="vSwing">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Vertical swing</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
